src/contexts/ApiContext.jsx

```diff
@@ -41,8 +40,21 @@ export const ApiProvider = (props) => {
       });
   };
 
+  const getCourse = () => {
+    axios
+      .get(`${backend}/courses`)
+      .then((res) => {
+        setCourse(res.data);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
```

Comment:
We *may* use `console.log` as error handlers in backend, but in frontend we want a 100% clean console before project end

Reply:
Fixed. i removed error handlers